### ISSUE TITLE
Issue installing f8-tagger via github

### ISSUE THREAD

**Issue by sara-02** [fabric8-analytics/fabric8-analytics-tagger#28]
I have a virtual env for python 3.5 and ran `python3 setup.py install` it is giving the following error:
```
writing top-level names to f8a_tagger.egg-info/top_level.txt
error: [Errno 13] Permission denied: 'f8a_tagger.egg-info/top_level.txt'
```
If I run it with `sudo` it gives the following error:
```
  File "setup.py", line 27, in _post_install
    import nltk
ImportError: No module named 'nltk'

```

**Comment by fridex** [comment]
I cannot reproduce this, not even in a container. BTW we also install tagger in core workers, this issue is not present there. How did you create virtual environment?

**Comment by sara-02** [comment]
@fridex 

**Steps:**
```
virtualenv --python=/usr/bin/python3.5 env
source env/bin/activate
python3 setup.py install
```

**Error:**
```
Traceback (most recent call last):
  File "setup.py", line 65, in <module>
    "Programming Language :: Python :: Implementation :: PyPy"
  File "/usr/lib64/python3.5/distutils/core.py", line 148, in setup
    dist.run_commands()
  File "/usr/lib64/python3.5/distutils/dist.py", line 955, in run_commands
    self.run_command(cmd)
  File "/usr/lib64/python3.5/distutils/dist.py", line 974, in run_command
    cmd_obj.run()
  File "setup.py", line 31, in run
    self.execute(_post_install, [], msg="preparing NLTK")
  File "/usr/lib64/python3.5/distutils/cmd.py", line 336, in execute
    util.execute(func, args, msg, dry_run=self.dry_run)
  File "/usr/lib64/python3.5/distutils/util.py", line 301, in execute
    func(*args)
  File "setup.py", line 27, in _post_install
    import nltk
ImportError: No module named 'nltk'

```
My default Python version is `Python 2.7.13` and I am on  `Fedora 25`

**Comment by fridex** [comment]
This should be fixed by #33. Feel free to reopen this issue if it persists.
